lib/fixup.rb

```diff
@@ -21,8 +27,28 @@ def require_gem(gem_name_and_require = nil, require_override = nil)
 end
 require_gem('highline/import')
 
+# All needed constants & variables should be defined here in case they
+# have not yet been loaded or fixup is being run standalone.
+
 CREW_VERBOSE = ARGV.intersect?(%w[-v --verbose]) unless defined?(CREW_VERBOSE)
 
+HOME = '/home/chronos/user' unless defined?(HOME)
+CREW_PREFIX = '/usr/local' unless defined?(CREW_PREFIX)
+CREW_LIB_PATH = File.join(CREW_PREFIX, 'lib/crew') unless defined?(CREW_LIB_PATH)
+CREW_CONFIG_PATH = File.join(CREW_PREFIX, 'etc/crew') unless defined?(CREW_CONFIG_PATH)
+CREW_META_PATH = File.join(CREW_CONFIG_PATH, 'meta') unless defined?(CREW_CONFIG_PATH)
+# via git log --reverse --oneline lib/const.rb | head -n 1
+CREW_CONST_GIT_COMMIT = '72d807aac' unless defined?(CREW_CONST_GIT_COMMIT)
```

Comment:
Why are we hardcoding a commit here if not available?

Reply:
That's just the first commit that lists const.rb ...
```
git/chromebrew$ git log --reverse --oneline lib/const.rb | head -n 5
72d807aac Refactoring crew
ef2a9302c Change default option for xz to use multi thread
dd3222416 Add crew -V|--version and -v|--verbose options
acb470333 Add CHROMEOS_RELEASE constant
33540a449 Add recursive build from source option to crew (#2499)
```

Reply:
(It's entirely silly...)